packages/neptune-extension/src/nbextension/components/Modal.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Jupyter from 'base/js/namespace';
+import dialog from 'base/js/dialog';
+
+const Modal = ({
+  children,
+}) => {
+  const ref = React.useRef(null);
+
+  if (ref.current === null) {
+    ref.current = document.createElement('div');
+
+    const modal = dialog.modal({
+      show: false,
+      title: 'Configure connection to Neptune',
+      notebook: Jupyter.notebook,
+      keyboard_manager: Jupyter.notebook.keyboard_manager,
+      body: ref.current,
+    });
+
+    modal.find('.neptune-popup-closer')
```

Comment:
Where this css  class name comes from?

Reply:
This has to be a leftover from nbextension code, for the moment I do not recall what is the exact effect of running this, I guess we can remove it for now and get to it back later.

Reply:
We are doing rewrite here so I would strip any code that is not needed.
Better to have clean code as a base for features rather thank think about why some code is here and there.